runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -301,14 +303,21 @@ public void committed(long epoch, Layout layout, boolean force)
                 })
                 .toArray(CompletableFuture[]::new);
 
+        // Make sure a quorum gets to commit.
         int timeouts = 0;
-        int responses = 0;
-        while (responses < commitList.length) {
-            // wait for someone to complete.
+        long wrongEpochRejected = 0L;
+        while (true) {
```

Comment:
Why spin here, shouldn't this be a best effort? the fault detector should periodically patch the management layout, commited layout and epochs

Reply:
As of now the fault detector does not do that and hence this patch with FIXMEs